pkg/ruler/manager_metrics_test.go

```diff
@@ -143,131 +143,35 @@ cortex_prometheus_rule_group_rules{rule_group="group_two",user="user3"} 100000
 func populateManager(base float64) *prometheus.Registry {
 	r := prometheus.NewRegistry()
 
-	metrics := newGroupMetrics(r)
+	metrics := rules.NewGroupMetrics(r)
```

Comment:
This seems like a pretty safe change - we assert on all the metrics that we actually use in the previous assertion.